go.mod

```diff
@@ -255,5 +255,3 @@ require (
 	lukechampine.com/blake3 v1.2.1 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
-
-replace github.com/livepeer/ai-worker => /home/ricks/development/livepeer/ai/ai-worker
```

Comment:
@eliteprox please ensure that the right version is used when https://github.com/livepeer/ai-worker/pull/163 is merged.